recipes/pyspeckit/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pyspeckit" %}
+{% set version = "0.1.21" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 80ffbd125f7b6e58c4019c17ad06d3d40e9d5052a83cbbd55513b2b9526cc581
+  patches:
+    - setup.py.patch
+    - setup.cfg.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - astropy
+    - matplotlib >=1.4
+    - numpy
+    - python
+
+test:
+  imports:
+    - examples
```

Comment:
Is `examples` supposed to be importable? It looks like something which is likely to conflict with other packages unless it is a namespace package or something similar.

If not, it doesn't need to be fixed for this PR but it might make sense to remove it from the tests.

Reply:
There is an examples folder but it does not need to be tested, have removed it.